README.md

```diff
@@ -1,36 +1,36 @@
-# Piwik Twig integration
+# Matomo Twig integration
 
-[![Build Status](https://travis-ci.org/mnapoli/PiwikTwigExtension.svg)](https://travis-ci.org/mnapoli/PiwikTwigExtension) [![Coverage Status](https://img.shields.io/coveralls/mnapoli/PiwikTwigExtension.svg)](https://coveralls.io/r/mnapoli/PiwikTwigExtension)
+[![Build Status](https://travis-ci.org/mnapoli/MatomoTwigExtension.svg)](https://travis-ci.org/mnapoli/MatomoTwigExtension) [![Coverage Status](https://img.shields.io/coveralls/mnapoli/MatomoTwigExtension.svg)](https://coveralls.io/r/mnapoli/MatomoTwigExtension)
 
-This library integrates [Piwik](http://piwik.org/) into [Twig](https://twig.symfony.com/).
+This library integrates [Matomo](http://matomo.org/) into [Twig](https://twig.symfony.com/).
 
 ## Installation
 
-    composer require "mnapoli/piwik-twig-extension"
+    composer require "mnapoli/matomo-twig-extension"
```

Comment:
I don't know if we should update the package name.
It'll require @mnapoli to create a new package, people will be lost, etc.

I think we should keep the same name but only rename classes, etc. as @tcitworld suggested.

I also don't have experience in renaming a composer package, maybe it's more easy than I though

Reply:
Hi! It is possible to rename, old package versions will still work, new versions will be available only under the new name. I'm not sure I'll have to create a new package, but I can do that if that helps. It's up to you, but I understand if you want to keep things as they are to avoid confusion.